fe/fe-core/src/main/java/com/starrocks/catalog/PrimitiveType.java

```diff
@@ -112,6 +113,25 @@
                     .addAll(STRING_TYPE_LIST)
                     .build();
 
+    private static final ImmutableSortedSet<String> VARIABLE_TYPE_SET =
```

Comment:
Why some use sort collector and some don't?

Reply:
It's not for sorting, but for case-insensitive.